book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -536,7 +536,7 @@ Create a new file in the `tmp` directory and close the container:
 
 Inspect the files in `tmp` on the host. The file `a.txt` was modified by the container process:
 ```console
-$ cat tmp/a.txt
+cat tmp/a.txt
 Hello
 World!
 ```
```

Comment:
Would be worth leaving it here too as there is output.

Reply:
Reverted - thank you!

https://github.com/the-turing-way/the-turing-way/blob/kw-jb2-formatting/book/website/reproducible-research/renv/renv-containers.md?plain=1#L539